app/javascript/components/molecules/ContentSection.js

```diff
@@ -39,7 +38,7 @@ ContentSection.Content = ({ children, borderBottom, paddingBottomNone, marginTop
   )
 }
 
-ContentSection.Sub = ({ title, description, borderBottom = true, children }) => (
+ContentSection.Sub = ({ title, description, borderBottom = false, children }) => (
```

Comment:
Not asking you to change anything, but one rule we followed at my last job that I liked is that optional boolean props _always_ default to false. I recommend we make that a general style rule across partners too

using this component for example, the two options for calling it with borderbottom look like:
```
// with a border
<ContentSection.Sub borderBottom />
```
or
```
// without a border
<ContentSection.Sub />
```

When the prop defaults to true your options look like:
```
// with a border
<ContentSection.Sub />
```
or
```
// without a border
<ContentSection.Sub  borderBottom={false} />
```

Which is just eeeever so slightly more noise since you can't use the "prop name defaults to true" shorthand.